libs/filamat/src/MaterialBuilder.cpp

```diff
@@ -941,10 +947,11 @@ MaterialBuilder& MaterialBuilder::useLegacyMorphing() noexcept {
 }
 
 Package MaterialBuilder::build(JobSystem& jobSystem) noexcept {
+    bool success;
     if (materialBuilderClients == 0) {
-        utils::slog.e << "Error: MaterialBuilder::init() must be called before build()."
-            << utils::io::endl;
+        slog.e << "Error: MaterialBuilder::init() must be called before build()." << io::endl;
         // Return an empty package to signal a failure to build the material.
+error:
```

Comment:
I think simply returning `Package::invalidPackage()` at each error is less bug-prone compared to a goto.